english/thinkofchile/thinkofchile.rtf

```diff
@@ -0,0 +1,9 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1404\cocoasubrtf340
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.